src/components/EditableField.jsx

```diff
@@ -97,24 +119,37 @@ const EditableField = (props) => {
   };
 
   return (
-    <Space>
-      {renderEditState()}
-      {
-        deleteEnabled
-          ? (
-            <Tooltip placement='bottom' title='Delete' mouseLeaveDelay={0}>
-              <Button size='small' shape='circle' icon={<DeleteOutlined />} onClick={deleteEditableField} />
-            </Tooltip>
-          ) : <></>
-      }
-    </Space>
+    <>
+      <Space direction='vertical'>
+        <Space>
+          {renderEditState()}
+          {
+            deleteEnabled
+              ? (
+                <Tooltip placement='bottom' title='Delete' mouseLeaveDelay={0}>
+                  <Button size='small' shape='circle' icon={<DeleteOutlined />} onClick={deleteEditableField} />
+                </Tooltip>
+              ) : <></>
+          }
+        </Space>
+        {!isValid ? (
+          <Text type='danger'>
+            {errorText}
+            {errorFunc(editedValue)}
+          </Text>
+        ) : <></>}
+      </Space>
+    </>
   );
 };
 
 EditableField.defaultProps = {
   onAfterSubmit: () => null,
   onAfterCancel: () => null,
   onDelete: () => null,
+  validationFunc: null,
+  errorFunc: () => null,
+  errorText: '',
```

Comment:
Do we actually need this `errorText` variable? I didn't see it set anywhere in the code

Reply:
`errorText` can be used together with `validationFunc` in the case where `validationFunc` does not return a value, and the `errorText` is quite simple. But yea, I think you're right, this can also be simply implemented as `errorFunc : () => 'Error message' `

Reply:
I'll fix this up

Reply:
sounds good!